tests/flex/test_wrap_executable.py

```diff
@@ -0,0 +1,22 @@
+import os
+from pyiron_base._tests import TestWithProject
+
+
+class TestExecutableContainer(TestWithProject):
+    def test_conda_environment_path(self):
+        python_version_step = self.project.wrap_executable(
+            job_name="pythonjobstep",
+            executable_str="python --version",
+            write_input_funct=None,
+            collect_output_funct=None,
+            input_dict=None,
+            conda_environment_path=None,
+            conda_environment_name=None,
+            input_file_lst=None,
+            execute_job=True,
+        )
+        self.assertTrue(python_version_step.status.finished)
+        self.assertEqual(
+            python_version_step.files.error_out,
```

Comment:
It would be great if the test case could actually produce some minimal output, because as it is this line doesn't actually test the new functionality but rather the file browser. Otherwise lgtm.

Reply:
I added a second test based on the example above.